common-rendering/src/editorialPalette.ts

```diff
@@ -781,6 +824,7 @@ const text = {
 	standfirstDark: textStandfirstDark,
 	standfirstLink: textStandfirstLink,
 	seriesTitle: textSeriesTitle,
+	pagination: textPagination,
```

Comment:
`editorialPalette` isn't showing the diff because it's too large, but I noticed this while we were mobbing - it looks like fields are in alphabetical order so it would be good to maintain the ordering in all objects and move `pagination` up (in `text`, `background`, `palette` etc)! Total nitpick, I know! :-)

Reply:
I think @mxdvl attempted something like this with ESLint?

Reply:
https://eslint.org/docs/rules/sort-keys
https://github.com/guardian/dotcom-rendering/pull/3907